test/DynamicExpresso.UnitTest/ConstructorTest.cs

```diff
@@ -179,6 +179,84 @@ public void Array_multi_dimension_constructor()
 			Assert.Throws<ParseException>(() => target.Parse("new int[,] { { 1 }, { 2 } }"));
 		}
 
+		[Test]
+		public void Ctor_NewDictionaryWithItems()
+		{
+			var target = new Interpreter();
+			target.Reference(typeof(System.Collections.Generic.Dictionary<,>));
+			var l = target.Eval<System.Collections.Generic.Dictionary<int, string>>("new Dictionary<int, string>(){{1, \"1\"}, {2, \"2\"}, {3, \"3\"}, {4, \"4\"}, {5, \"5\"}}");
+			Assert.AreEqual(5, l.Count);
+			for (int i = 0; i < l.Count; ++i)
+			{
+				Assert.AreEqual(i + 1 + "", l[i + 1]);
+			}
+		}
+
+		[Test]
+		public void Ctor_NewMyClassWithItems()
+		{
+			var target = new Interpreter();
+			target.Reference(typeof(MyClassAdder));
+			var l = target.Eval<MyClassAdder>("new MyClassAdder(){{ 1, 2, 3, 4, 5},{\"6\" },7	}.Add(true)");
+			Assert.AreEqual(5, l.MyArr.Length);
+			for (int i = 0; i < l.MyArr.Length; ++i)
+			{
+				Assert.AreEqual(i + 1, l.MyArr[i]);
+			}
+			Assert.AreEqual("6", l.StrProp);
+			Assert.AreEqual(7, l.IntField);
+		}
+
+
+		[Test]
+		public void Ctor_NewMyClassWithCross()
+		{
+			string StrProp = "";
```

Comment:
Please always use var, and use `string.Empty`  insteaf of `""`

Reply:
Incorporated in latest update